src/pages/metapackages/commerce/index.md

```diff
@@ -52,7 +52,19 @@ At this point, you should see symlinks for all the `pwa-commerce` modules inside
 -  Run a Magento installation with additional modules.
 -  Develop locally using the standard git workflow.
 
-You may need to ensure that there are no `Magento_PWA*` modules listed as `enabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+You may need to ensure that there are no `Magento_*Pwa` modules listed as `disabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+
+Note: You should expect to see the following modules
+
+    Magento_ContactGraphQlPwa
```

Comment:
Remove indentation and wrap in code fences with bash language specified after first fence

```
```bash
```

Reply:
I think `terminal` is a better code fence. This is command output.

Reply:
Good cal @keharper. I didn't think "terminal" was supported, but it is.